3.5/Dockerfile.fedora

```diff
@@ -15,7 +15,7 @@ LABEL BZComponent="$NAME" \
       Release="$RELEASE.$DISTTAG" \
       Architecture="$ARCH" \
       Usage="s2i build file:///your/app 25/python your-app" \
-      Help="help.1"
+      Help="inside the container run: $ man help"
```

Comment:
Oh! I didn't know about `atomic help <img>`, which caused my confusion. We could mention it in `Help` as well:

    Help="https://github.com/sclorg/s2i-python-container/tree/master/3.5 or run: $ atomic help <img>"